templates/about.html

```diff
@@ -1,6 +1,32 @@
 {% extends "layout.html" %}
 {% block content %}
 
-about
+<head>
+<title> ABOUT </title>
+<h1> About Mars Rocks </h1>
+
+<style>
+h2 {
+text-align: center;
+font-family: Impact;
+color: burgundy
+}
+
+</style>
+
+</head>
+
+<body>
+<h2><strong>Welcome to the Mars Rocks!</strong></h2>
```

Comment:
delete lines 4 <head>  to lines 19  <body> . That code is already in the layout.html.  Layout.html is the base page, and all the other templates are embedded inside layout.html.